hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/calculator/ContractCallFeeCalculator.java

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
+package com.hedera.node.app.service.contract.impl.calculator;
+
+import com.hedera.hapi.node.transaction.TransactionBody;
+import com.hedera.node.app.spi.fees.FeeContext;
+import com.hedera.node.app.spi.fees.ServiceFeeCalculator;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import org.hiero.hapi.fees.FeeResult;
+import org.hiero.hapi.support.fees.FeeSchedule;
+
+public class ContractCallFeeCalculator implements ServiceFeeCalculator {
+    @Override
+    public void accumulateServiceFee(
+            @NonNull final TransactionBody txnBody,
+            @Nullable final FeeContext feeContext,
+            @NonNull final FeeResult feeResult,
+            @NonNull final FeeSchedule feeSchedule) {
+        // we clear the node and network fee previously set by SimpleFeeCalculatorImpl,
+        // as contract call is paid only in gas
```

Comment:
@tinker-michaelj is this correct? We don't pay the network or node anything for contract calls?